store/tikv/coprocessor.go

```diff
@@ -628,6 +630,58 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
 	return worker.handleCopResponse(bo, resp.Cop, task, ch, nil)
 }
 
+const (
+	minLogBackoffTime       = 100
+	minLogKVProcessTime     = 100
+	minLogKVWaitTime        = 200
+	minLogScanProcessedInfo = 100000
+	minLogScanTotalInfo     = 200000
+)
+
+func (worker *copIteratorWorker) logTimeCopTask(costTime time.Duration, task *copTask, bo *Backoffer, resp *tikvrpc.Response) {
+	logStr := fmt.Sprintf("[TIME_COP_TASK] resp_time:%s txn_start_ts:%d region_id:%d store_addr:%s", costTime, worker.req.StartTs, task.region.id, task.storeAddr)
+	if bo.totalSleep > minLogBackoffTime {
+		backoffTypes := strings.Replace(fmt.Sprintf("%v", bo.types), " ", ",", -1)
+		logStr += fmt.Sprintf(" backoff_ms:%d backoff_types:%s", bo.totalSleep, backoffTypes)
+	}
+	var detail *kvrpcpb.ExecDetails
+	if task.cmdType == tikvrpc.CmdCopStream {
+		detail = resp.CopStream.ExecDetails
+	} else {
+		detail = resp.Cop.ExecDetails
+	}
+	if detail != nil {
+		if detail.HandleTime != nil {
+			processMs := detail.HandleTime.ProcessMs
+			waitMs := detail.HandleTime.WaitMs
+			if processMs > minLogKVProcessTime {
+				logStr += fmt.Sprintf(" kv_process_ms:%d", processMs)
+			}
+			if waitMs > minLogKVWaitTime {
+				logStr += fmt.Sprintf(" kv_wait_ms:%d", waitMs)
+			}
+		}
+		if detail.ScanDetail != nil {
+			logStr = appendScanDetail(logStr, "write", detail.ScanDetail.Write)
+			logStr = appendScanDetail(logStr, "data", detail.ScanDetail.Data)
+			logStr = appendScanDetail(logStr, "lock", detail.ScanDetail.Lock)
+		}
+	}
+	log.Info(logStr)
+}
+
+func appendScanDetail(logStr string, columnFamily string, scanInfo *kvrpcpb.ScanInfo) string {
+	if scanInfo != nil {
+		if scanInfo.Total > minLogScanTotalInfo {
```

Comment:
Why we need to check this?

Reply:
I think we should not log messages that is not interesting.